brain/task_executor.py

```
@@ -943,94 +989,126 @@ async def _execute_user_plugin(self, task_id: str, up_decision: Any) -> TaskResu
+                ctx_obj = safe_args.get("_ctx")
+                if not isinstance(ctx_obj, dict):
+                    ctx_obj = {}
+                if lanlan_name and "lanlan_name" not in ctx_obj:
+                    ctx_obj["lanlan_name"] = lanlan_name
+                # 添加 conversation_id，用于关联触发事件和对话上下文
+                if conversation_id:
+                    ctx_obj["conversation_id"] = conversation_id
+                if ctx_obj:
+                    safe_args["_ctx"] = ctx_obj
+            except Exception:
+                pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)